layouts/partials/main/blog-meta.html

```diff
@@ -9,4 +13,9 @@
       <img class="rounded-circle w-auto mx-1 lazyload blur-up" src="{{ $imageLq.RelPermalink }}" data-src="{{ $image.RelPermalink }}" alt="{{ .Title }}" width="30" height="30">
   {{ end -}}
 {{ end -}}
-<a class="stretched-link position-relative" href="{{ "/contributors/" | relLangURL }}{{ . | urlize }}/">{{ . }}</a>{{ end -}}{{ end -}}&nbsp;&hyphen;&nbsp;<strong>{{ .ReadingTime -}}&nbsp;min read</strong></small><p>
+<a class="stretched-link position-relative" href="{{ "/contributors/" | relLangURL }}{{ . | urlize }}/">{{ . }}</a>{{ end -}}{{ end -}}&nbsp;&hyphen;&nbsp;<strong>{{ .ReadingTime -}}{{ if  i18n "reading-time" }}
+    {{i18n "reading-time"}}
```

Comment:
Instead of `{{ if something }}{{ something }}` use `{{ with something }}{{ . }}`, see https://gohugo.io/templates/introduction/#conditionals

Reply:
Done, thx for the feedback pretty new to hugo

Reply:
> Done

Not quite. `with` [rebinds the context (`.`) within its scope](https://gohugo.io/functions/go-template/with/), i.e. inside the `with` statement, the dot (`.`) becomes what `with` referenced (`i18n "reading-time"` in this particular example).

Replace `{{i18n "reading-time"}}` with `{{ . }}`. 🙂

Reply:
Adjusted the statement. Thx for link to docs of `with`. 
From a purely technical point of view, it would have worked the same way, wouldn't it? So it's more best practice?
Again thx for the feedback :)